### ISSUE TITLE
Zonemaster crasches in DNSSEC13

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-engine#727]
When running Zonemaster (version v2019.2), it crashed during test av [DNSSEC13]. Here as running `zonemaster-cli` but it has been verified in Zonemaster-Backend too.

```
$ zonemaster-cli ip.se --no-ipv6 --test DNSSEC/dnssec13 --show_module --raw --level INFO
   0.01 INFO      SYSTEM       SYSTEM:GLOBAL_VERSION version=v3.0.3
Use of uninitialized value in subroutine entry at /usr/local/lib/x86_64-linux-gnu/perl/5.26.1/Zonemaster/LDNS/RR/RRSIG.pm line 12.
   3.79 ERROR     DNSSEC       DNSSEC:RRSIG_BROKEN address=162.219.54.128; keytag=5134; ns=ns1.anycast.iis.se; rrtype=DNSKEY
Segmentation fault (core dumped)
```
(Disregard `--no-ipv6`.)

The consequence for `zonemaster-cli` is that the test breaks and ends with "core dump".

The consequence for Zonemaster-Backend is that the specific testagent process that runs the test will crash, but the primary `testagent` process will continue. The test will never be completed, but neither the `testagent` function nor the `RPC-API` function will be affected.

[DNSSEC13]: https://github.com/zonemaster/zonemaster/blob/704829668719ff61ed7d99da597c1f4241bb4c8a/docs/specifications/tests/DNSSEC-TP/dnssec13.md

**Comment by matsduf** [comment]
Resolved by #728.

**Comment by matsduf** [comment]
The bug was triggered when the zone had a RRSIG referring to absent DNSKEY.
